From 5862361928d626823f97203e45257fc87444b40a Mon Sep 17 00:00:00 2001
From: Jens Lang <j.lang@phytec.de>
Date: Thu, 25 Feb 2021 11:42:04 +0000
Subject: [zeus-next v1] dynamic-layers: imx-atf: disable uart4 assigned to arm-cortex m4

In normal case uart4 is assigned to arm-cortex m4 as debug interface.
For tauri l boards it's neccesarry to unassign it.

Signed-off-by: Jens Lang <j.lang@phytec.de>
---
 plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c b/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
index 0491928..8e29440 100644
--- a/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
+++ b/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
@@ -58,7 +58,8 @@ static const struct imx_rdc_cfg rdc[] = {
 	RDC_MDAn(RDC_MDA_M4, DID1),
 
 	/* peripherals domain permission */
-	RDC_PDAPn(RDC_PDAP_UART4, D1R | D1W),
+	/* RDC_PDAPn(RDC_PDAP_UART4, D1R | D1W), */
+	RDC_PDAPn(RDC_PDAP_UART4, D0R | D0W),
 	RDC_PDAPn(RDC_PDAP_UART2, D0R | D0W),
 
 	/* memory region */
-- 
2.7.4

